### ISSUE TITLE
Selected date is not shown in the input field

### ISSUE THREAD

**Issue by cavishek39** [wojtekmaj/react-date-picker#355]
When, I clicked on a different date other than the current date, that selected date is not showing in the date input field.

**Comment by VictorBaba** [comment]
I have the same issue. Looks like is more a problem of Safari, because on Chrome everything is working.

**Comment by ron51** [comment]
Yes, I am facing the same issue. @VictorBaba @cavishek39 were you able to find an alternate solution?

**Comment by cavishek39** [comment]
I've used material date picker now.

**Comment by wojtekmaj** [comment]
Is there an easy way to reproduce this? Specific browser, browser version, specific code?

**Comment by HPieters** [comment]
@wojtekmaj I can reproduce this issue on Safari when I use react-date-picker in a modal. It works just fine if I use it outside of a modal. The strange thing is that if you wrap DatePicker with a div and an onClick to control isOpen the area that is covered by the div (and calendar that opens) will actually fill the input.

Reproduction here: [https://codesandbox.io/s/react-date-picker-modal-example-71sx5 ](https://codesandbox.io/s/react-date-picker-modal-example-71sx5).

Date picker in modal won't work in Safari (for me) and it will work outside the modal. Works in Chrome / Brave.

**Comment by sohaib-baltoro** [comment]
Is there any update on this issue? I am facing the same issue on safari
